ansible.cfg

```diff
@@ -0,0 +1,6 @@
+[defaults]
+ask_sudo_pass = True
+transport = ssh
+
+[ssh_connection]
+ssh_args = -o ForwardAgent=yes
```

Comment:
@tkellen the config file is being used here because of the Forward Agent option. So we can drop the `
deploy_key_path` variable.